tests/integration/bootstrap/FeatureContext.php

```diff
@@ -80,4 +80,18 @@ public function disableDataImport()
 		$config = \Kohana::$config->load('features');
 		$config->set('data-import.enabled', false);
 	}
+
+	/** @BeforeScenario @cdnEnabled */
+	public function enableCdn()
```

Comment:
I removed this and now the checks have failed. @rjmackay Can we touch base on this?